Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.yml

```diff
@@ -1042,11 +1075,19 @@ script:
     outputs:
     - contextPath: Threatgrid.SearchResult
       description: Result for the searched query
-    description: Advanced search that allows searching URLS, Submissions, Samples
-      etc...
-  
+    description: Advanced search that allows searching URLS, Submissions, Samples etc...
+
+    deprecated: true
   runonce: false
   dockerimage: demisto/python3:3.10.9.42476
 tests:
-- ThreatGridTest
+- No tests (deprecated)
```

Comment:
Is this a necessity for deprecation? AFAIK it's not

Reply:
So reset it?

Reply:
yes, please.